lib/portage/package/ebuild/_config/special_env_vars.py

```diff
@@ -348,9 +349,10 @@
 ]
 
 # No longer supported variables
-environ_filter += ["SYNC"]
+environ_filter_temp += ["SYNC"]
 
-environ_filter = frozenset(environ_filter)
+environ_filter = frozenset(environ_filter_temp)
+del environ_filter_temp
```

Comment:
Again, this temporary variable just clutters the code. I don't like this.

Reply:
Removed this one too.